src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementTranslator.java

```diff
@@ -114,4 +133,12 @@ private AttachmentId getImageAttachmentId(OrganizationId organizationId, Integer
     return kuntaApiId;
   }
   
+  private OffsetDateTime toOffsetDateTime(LocalDateTime date) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "toOffsetDateTime" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)